extension/page-detect.js

```diff
@@ -1,7 +1,5 @@
 window.pageDetect = (() => {
-	const isGistCheck = location.hostname === 'gist.github.com';
-
-	const isGist = () => isGistCheck;
+	const isGist = () => location.hostname === 'gist.github.com';
```

Comment:
@DrewML I did this like that, because `isGist()` is called for every `isRepo()` call which is called for every other check. But `location.hostname` can't change without invalidating the JS context, so it's safe to check it only once.